bonfire/config.py

```diff
@@ -45,7 +45,10 @@
 QONTRACT_PASSWORD = os.getenv("QONTRACT_PASSWORD", APP_INTERFACE_PASSWORD or None)
 QONTRACT_TOKEN = os.getenv("QONTRACT_TOKEN")
 
-BASE_NAMESPACE_NAME = os.getenv("BASE_NAMESPACE_NAME", "ephemeral-base")
+BASE_NAMESPACE_PATH = os.getenv(
+    "BASE_NAMESPACE_PATH",
+    "/services/insights/ephemeral/namespaces/ephemeral-base.yml",
```

Comment:
Why did you need to change this?

Reply:
This was related to: 

> I also found a bug related to how we match namespaces to an environment. Some namespaces now have the same name across environments. So now, we match on the full namespace path instead of just the name.

Reply:
As an example, `frontends` and `frontends-base` exist in both stage and prod envs. But even for ephemeral-base, if you look for all namespaces named 'ephemeral-base' in app-interface configs, you'll actually find there's more than 1 :) There is one tied to the ephemeral cluster, and one tied to the dev cluster.

Reply:
OK that makes sense! Thanks for the clarification.